application-accelerator/provenance.hbs.md

```diff
@@ -21,3 +21,54 @@ project:
 
 For more information about the structure of the file and how to enable application bootstrapping
 provenance, see [Provenance transform](creating-accelerators/transforms/provenance.hbs.md).
+
+## <a id="amr"></a> Integration with AMR
+
+Application Accelerator integrates with [Artifact Metadata Repository](../scst-store/overview.hbs.md) (AMR).
+
+When you generate an application with an accelerator, an event that contains the same information captured by the `Provenance` transform is sent to the AMR store.
+
+The relevant data saved to AMR are:
```

Comment:
Is there something missing here?
What is being referred to here?

Reply:
the relevant data saved to AMR is referring to the two headers below
- AppAcceleratorRuns
- AppAcceleratorFragments

please let me know if I can format this better